src/SDKs/CognitiveServices/dataPlane/Search/Search/Microsoft.Azure.CognitiveServices.Search.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
-    <PackageId>Microsoft.CognitiveServices.Search</PackageId>
+    <PackageId>Microsoft.Azure.CognitiveServices.Search</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Search APIs.</Description>
     <VersionPrefix>1.0.0-preview</VersionPrefix>
     <DefineConstants>$(DefineConstants);CODESIGN</DefineConstants>
```

Comment:
These constants are not needed. If you need it, can you explain the purpose of these constants.

Reply:
I can't, I'm copying the structure of the Vision / Face csprojs since we want the exact same behavior / publishing story. This was recommended by David Li

Reply:
This is my reference: https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Vision/Vision/Microsoft.Azure.CognitiveServices.Vision.csproj

Reply:
synced offline, thanks for the clarification